src/box/lua/console.lua

```diff
@@ -617,13 +617,16 @@ local function local_read(self)
     local prompt = self.prompt
     while true do
         local delim = self.delimiter
-        local line = internal.readline({
+        local line, discard_buffer = internal.readline({
             prompt = prompt.. "> ",
             completion = self.ac and self.completion or nil
         })
         if not line then
             return nil
         end
+        if discard_buffer == true then
```

Comment:
Nit: A boolean in an if-condition is usually not checked against equality with true/false.